source/extensions/transport_sockets/tls/cert_validator/default_validator.cc

```diff
@@ -195,6 +195,7 @@ int DefaultCertValidator::doVerifyCertChain(
 
     if (ret <= 0) {
       stats_.fail_verify_error_.inc();
+      ENVOY_LOG(debug, "{}", Utility::getX509VerificationErrorInfo(store_ctx));
```

Comment:
I would like you to add this line to SPIFFE validator as well: https://github.com/envoyproxy/envoy/blob/main/source/extensions/transport_sockets/tls/cert_validator/spiffe/spiffe_validator.cc#L162

Reply:
nvm -- I will take care of this later (seem a bit complex).

Reply:
thanks!